src/main/webapp/scripts/script-session.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
```

Comment:
Rename this file to session-script.js, same thing for the test. session-script-test.js. So that its consistent with session-style. js